locale/templates/LC_MESSAGES/django.pot

```diff
@@ -7129,58 +7142,86 @@ msgid ""
 "any later version."
 msgstr ""
 
-#: src/olympia/templates/footer.html:3
-msgid "get to know <b>add-ons</b>"
+#: src/olympia/templates/header_title.html:3
+msgid "Return to the {0} Add-ons homepage"
+msgstr ""
+
+#: src/olympia/templates/header_title.html:5
+#: src/olympia/templates/photon-footer.html:8
+msgid "Add-ons"
 msgstr ""
 
-#: src/olympia/templates/footer.html:4
+#: src/olympia/templates/photon-footer.html:4
+msgid "Go to Mozilla's homepage"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:11
 msgid "About"
 msgstr ""
 
-#: src/olympia/templates/footer.html:7
+#: src/olympia/templates/photon-footer.html:12
+msgid "Firefox Add-ons Blog"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:16
+msgid "Community Blog"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:17
 msgid "Forum"
 msgstr ""
 
-#: src/olympia/templates/footer.html:13
-msgid "Download Firefox"
+#: src/olympia/templates/photon-footer.html:19
+msgid "Review Guide"
 msgstr ""
 
-#: src/olympia/templates/footer.html:14
-msgid "Android Browser"
+#: src/olympia/templates/photon-footer.html:24
+#: src/olympia/templates/photon-footer.html:37
+msgid "Browsers"
 msgstr ""
 
-#: src/olympia/templates/footer.html:15
-msgid "iOS Browser"
+#: src/olympia/templates/photon-footer.html:27
+msgid "Desktop"
 msgstr ""
 
-#: src/olympia/templates/footer.html:16
-msgid "Focus Browser"
+#: src/olympia/templates/photon-footer.html:29
+msgid "Enterprise"
 msgstr ""
 
-#: src/olympia/templates/footer.html:17
-msgid "Desktop Browser"
+#: src/olympia/templates/photon-footer.html:38
+msgid "VPN"
 msgstr ""
 
-#: src/olympia/templates/footer.html:18
-msgid "Beta, Nightly, Developer Edition"
+#: src/olympia/templates/photon-footer.html:39
+msgid "Relay"
 msgstr ""
 
-#: src/olympia/templates/footer.html:25
-#: src/olympia/templates/includes/lang_switcher.html:2
-msgid "Other languages"
+#: src/olympia/templates/photon-footer.html:40
+msgid "Monitor"
 msgstr ""
 
-#: src/olympia/templates/footer.html:36
-#: src/olympia/templates/includes/lang_switcher.html:10
-msgid "Go"
+#: src/olympia/templates/photon-footer.html:41
+msgid "Pocket"
 msgstr ""
 
-#: src/olympia/templates/header_title.html:3
-msgid "Return to the {0} Add-ons homepage"
+#: src/olympia/templates/photon-footer.html:52
+msgid "Privacy"
 msgstr ""
 
-#: src/olympia/templates/header_title.html:5
-msgid "Add-ons"
+#: src/olympia/templates/photon-footer.html:53
+msgid "Cookies"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:54
+msgid "Legal"
+msgstr ""
+
+#: src/olympia/templates/photon-footer.html:57
+msgid ""
+"Except where otherwise <a href=\"https://www.mozilla.org/en-"
```

Comment:
Never put `en-US` in links to mozilla.org or SUMO, let the website manage the locale negotiation.
https://www.mozilla.org/about/legal/

Reply:
:+1: I just happened to notice this footer wasn't translated but I didn't double-check the links when adding the gettext tag...